pkg/cmd/cmd.go

```diff
@@ -31,14 +35,16 @@ type patchArgs struct {
 	push          bool
 	platform      []string
 	loader        string
+	configFile    string
 }
 
 func NewPatchCmd() *cobra.Command {
 	ua := patchArgs{}
 	patchCmd := &cobra.Command{
-		Use:     "patch",
-		Short:   "Patch container images with upgrade packages specified by a vulnerability report",
-		Example: "copa patch -i images/python:3.7-alpine -r trivy.json -t 3.7-alpine-patched",
+		Use:   "patch",
+		Short: "Patch container images with upgrade packages specified by a vulnerability report or by comprehensive update",
```

Comment:
since this is the initial scope: 

```suggestion
		Short: "Patch container images with upgrade packages specified by comprehensive update",
```

Reply:
@sozercan but this short is for the patch command as a whole right? not just bulk image patch. maybe we can specify comprehensive update for bulk.